unit_tests/matrix_test/matrix_test.cpp

```diff
@@ -762,6 +762,24 @@ void LookAt_Test(const T& precision) {
 }
 TEST_SCALAR_F(LookAt, FLOAT_PRECISION, kLookAtDoublePrecision);
 
+// Test UnProject calculation.
+TEST_F(MatrixTests, UnProjectTest) {
+  mathfu::Matrix<float, 4, 4> modelView = mathfu::Matrix<float, 4, 4>(-1, 0,  0, 0,
+                                                                      0, 1,   0, 0,
+                                                                      0, 0,  -1, 0,
+                                                                      0, 0, -10, 1);
+  mathfu::Matrix<float, 4, 4> projection = mathfu::Matrix<float, 4, 4> (1.81066,          0,            0,  0,
+                                                                              0, 2.41421342,            0,  0,
+                                                                              0,          0,  -1.00001991, -1,
+                                                                              0,          0, -0.200001985,  0);
```

Comment:
style: could you wrap lines longer than 80 characters?

Also, if you want the formatting preserved you'll need to wrap these lines in

// clang-format off
...
// clang-format on

e.g

// clang-format off
mathfu::Matrix<float, 4, 4> modelView =
    mathfu::Matrix<float, 4, 4>(-1, 0,   0, 0,
                                 0, 1,   0, 0,
                                 0, 0,  -1, 0,
                                 0, 0, -10, 1);
// clang-format on